src/main/java/gyro/google/kms/CryptoKeyVersionResource.java

```diff
@@ -0,0 +1,143 @@
+package gyro.google.kms;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import com.google.cloud.kms.v1.CryptoKeyName;
+import com.google.cloud.kms.v1.CryptoKeyVersion;
+import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.protobuf.FieldMask;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Id;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+import gyro.google.GoogleResource;
+
+/**
+ * Create a new version for the crypto key.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    google::crypto-key-version crypto-key-version-example
+ *        crypto-key: $(google::crypto-key example-crypto-key)
+ *        state: ENABLED
+ *    end
+ */
+@Type("crypto-key-version")
+public class CryptoKeyVersionResource extends GoogleResource implements Copyable<CryptoKeyVersion> {
+
+    private CryptoKeyResource cryptoKey;
+    private CryptoKeyVersion.CryptoKeyVersionState state;
```

Comment:
If imported `import com.google.cloud.kms.v1.CryptoKeyVersion;`
```suggestion
    private CryptoKeyVersionState state;
```